opentelemetry-instrumentation/src/opentelemetry/instrumentation/auto_instrumentation/sitecustomize.py

```diff
@@ -28,9 +28,11 @@
 
 def initialize():
     # prevents auto-instrumentation of subprocesses if code execs another python process
-    environ["PYTHONPATH"] = _python_path_without_directory(
-        environ["PYTHONPATH"], dirname(abspath(__file__)), pathsep
-    )
+
+    if environ.get("PYTHONPATH") is not None:
```

Comment:
"if "PYTHONPATH" in environ" is the norm

Reply:
In many environment PYTHONPATH isn't set by default, this cause many error to start quickly to use the lib.